compiler/ballerina-lang/src/main/java/io/ballerina/projects/CompilationOptions.java

```diff
@@ -96,6 +98,10 @@ public boolean withCodeGenerators() {
         return toBooleanDefaultIfNull(this.withCodeGenerators);
     }
 
+    public Boolean getConfigSchemaGen() {
```

Comment:
```suggestion
    public Boolean configSchemaGen() {
```
Seems we don't use the prefix `get` for getter methods in this class.